python/core/auto_generated/qgsvectorlayer.sip.in

```diff
@@ -2279,6 +2279,12 @@ Configuration and logic to apply automatically on any edit happening on this lay
 
 
 
+    QgsStoredExpressionManager *storedFilterExpressions();
```

Comment:
My idea was to name it like this, because in future maybe will follow: `storedDefaultValueExpressions(QgsStoredExpressionManager::DefaultValue)` or `storedConstraintsExpressions(QgsStoredExpressionManager::Constraints)` 

This could be done with the tags as well. Advantage of the usage of the tag could be that we can combine it better: here use constraint - and filter expressions, there use defaulvalues and constrainexpressions etc. But not sure if we need it somewhere like this. And disadvantage is that everything is in one big list. Solved as above we have seperate lists for all kinds of stored expressions.

Btw. if I do not use at all the tags in this implementation, should I remove it?

Reply:
Is this method affected by tags? I think there will always be one manager per layer and this one manager handles the tags (or categories).

Reply:
Hm... so you prefer to have always one single manager containing several lists (defaultvalueexpressions, filterexpressions etc.) or one list (containing everything) but anyway categorized by the tags to identify if filter, defaultvalues etc..? 

I liked it to have several managers for several use cases, but I'm open...

Reply:
I'd prefer to have one manager to organize the expressions. The QgsVectorLayer api is crowded enough already.